consoles/sshVirtsh.pm

```diff
@@ -517,14 +517,17 @@ __END"
 }
 
 sub attach_to_running {
-    my ($self, $name) = @_;
+    my ($self, $args) = @_;
 
+    my $name = $args->{name};
```

Comment:
that sounds like you changing API. You can be backward compatible in checking ref($args). If it's undefined $args is $name